runtime/service/npud/CMakeLists.txt

```diff
@@ -34,5 +35,7 @@ target_include_directories(npud_dbus PUBLIC ${GIO_UNIX_2.0_INCLUDE_DIRS})
 target_link_libraries(npud_dbus PRIVATE ${GIO2.0_LIBRARIES})
 target_link_libraries(npud_dbus PRIVATE ${GIO_UNIX_2.0_LIBRARIES})
 
+install(FILES ${DBUS_CONFIG_FILE} DESTINATION bin)
```

Comment:
This enables test w/o system.d install ?

Reply:
No, it just copy to binary directory. To run `npud`, this file should be installed in `/usr/share/dbus/system.d/` directory...

Reply:
Then, is `bin` used as temporary place? IMO, install directory `bin` is not proper place.

Reply:
Where is a good place to put config file? 👀

Reply:
I don't know. But maybe not in `bin` and `lib`.

Reply:
@jyoungyun Do you have any idea to enable test on local environment ? Then, you can follow the location of conf.

Reply:
@chunseoklee To enable this test, it should be located to `/usr/share/dbus/system.d/` directory in local environment too...

@hseok-oh @chunseoklee 

How about using `share` directory because this file will be located under the `share` directory finally.

* Install it the `Product/[*].[debug/release]/share/`
* Install it the `Product/[*].[debug/releaes]/share/dbus/system.d`

Reply:
It is OK to me(shared).

Reply:
OK